numpy/_core/code_generators/genapi.py

```diff
@@ -465,12 +465,11 @@ def _key(x):
     return sorted(o, key=_key)
 
 def merge_api_dicts(dicts):
-    ret = {}
-    for d in dicts:
-        for k, v in d.items():
-            ret[k] = v
-
-    return ret
+    return {
```

Comment:
I don't think this is an improvement.

Reply:
List and dict comprehensions are much faster and very simple Python. What exactly don't you like? Having code after return instead of an intermediate variable? If so, it can easily be changed to:
```
    ret = {
        k: v
        for d in dicts
        for k, v in d.items()
    }
    return ret
```